### ISSUE TITLE
Getting some weird output ok ?? 36 ok ?? 36 ok ?? 36

### ISSUE THREAD

**Issue by limonte** [stylelint/stylelint#4146]
> Clearly describe the bug

I'm getting some weird output when running `stylelint src/**/*.scss`:

```
ok
?? 36
ok
?? 36
ok
?? 36
ok
?? 36
```

Live demo: https://travis-ci.org/sweetalert2/sweetalert2/builds/554841887

> What stylelint configuration is needed to reproduce the bug?

https://github.com/sweetalert2/stylelint-config/blob/master/index.js

> Which version of stylelint are you using?

`10.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

`stylelint src/**/*.scss`

> What did you expect to happen?

No weird output.

### Steps to reproduce 

1. `git clone https://github.com/sweetalert2/sweetalert2.git`
2. `cd sweetalert2`
2. `yarn install`
3. `./node_modules/.bin/stylelint src/**/*.scss`

OS: Linux, 
node version: 12.5.0

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

I was narrowed down to single `calc`:

```scss
/* test.scss */
body {
  max-width: calc(100% * 2);
}
```

Output:

```
?? 42
ok
```

Input:

```scss
/* test.scss */
body {
  max-width: calc(100% - #{$swal2-container-padding} * 2);
}
```

Output:

```
?? 45
ok
?? 35
?? 36
ok
```

When I disabled [`stylelint-csstree-validator`](https://github.com/csstree/stylelint-validator) in [config](https://github.com/sweetalert2/stylelint-config/blob/11a64300c2fd381820d1fca47fd4e3a980e8590f/index.js#L7) strange output disappeared. I think issue in a plugin.

**Comment by limonte** [comment]
Thank you so much @hudochenkov for the prompt investigation and reply, I opened the issue in `stylelint-csstree-validator`: https://github.com/csstree/stylelint-validator/issues/18

Feel free to close this one, or I'll close it when the root of the issue will be caught.
